README.md

```diff
@@ -11,4 +11,73 @@ However, if exceptions occur frequently, consider adjusting guidelines (by creat
 
 ## Table of contents
 
-// TODO
+- [Git](#git)
+  - [Workflow](#workflow)
+  - [Commit messages](#commit-messages)
+
+## [Git](#git)
+
+### [Workflow](#workflow)
+
+1. Create a feature branch  
+
+    - Prefix branch name with `feature/` or `fix/` when applicable; otherwise, prefix with your initials, e.g. `tk/`
```

Comment:
Please no branches with initials :D Also, `refactor/` is sometimes used as well :)

Reply:
Why not? Guys, let's be constructive and provide feedback instead of emotions or blank statements.

Reason for prefixes is to avoid clashes which may happen sometimes (from practice). I see nothing wrong here.

Reply:
Oh, that's why - I actually didn't have any painful experience with branch clashes. On the other hand - these initials feels like branch ownership, which might discourage collaboration, so I would suggest to use simply `feature/fix/refactor` or nothing in rare cases.